packages/web/src/components/data-entry/TextAreaV2.tsx

```diff
@@ -0,0 +1,108 @@
+import { useRef, useCallback, useEffect, ComponentPropsWithoutRef } from 'react'
```

Comment:
@raymondjacobson / @dylanjeffers should we add TextArea and other inputs to stems?